core/services/chainlink/relayer_chain_interoperators_test.go

```diff
@@ -317,10 +322,13 @@ func TestCoreRelayerChainInteroperators(t *testing.T) {
 				{Network: relay.Cosmos, ChainID: relay.ChainID(cosmosChainID1)},
 				{Network: relay.Cosmos, ChainID: relay.ChainID(cosmosChainID2)},
 			},
+
+			expectedRelayerNetworks: map[relay.Network]struct{}{relay.EVM: {}, relay.Cosmos: {}, relay.Solana: {}, relay.StarkNet: {}},
 		},
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
+			tt := tt
```

Comment:
Doesn't the linter still flag the `tt.name` usage? Normally this is before the `t.Run` call.

Reply:
no, it doesn't. maybe because there are no pointers accessed before the reassignment? anyway i changed to be first